nested/track.json

```diff
@@ -0,0 +1,52 @@
+{
+	"short-description": "Nested query benchmark using up to 11,203,029 questions from StackOverflow",
+	"description": "This test indexes and queries up to 11,203,0299 questions from StackOverflow (total 3.64 GB json)",
+	"data-url": "http://benchmarks.elasticsearch.org.s3.amazonaws.com/corpora/nested",
+	"indices": [{
+		"name": "sonested",
+		"types": [{
+			"name": "question",
+			"mapping": "mappings.json",
+			"documents": "documents.json.bz2",
+			"document-count": 11203029,
+			"compressed-bytes": 695293381,
+			"uncompressed-bytes": 3637747670
+		}]
+	}],
+	"operations": [{
+		"name": "index-append",
+		"operation-type": "index",
+		"bulk-size": 500
+	}, {
+		"name": "force-merge",
+		"operation-type": "force-merge"
+	},  {
+		"name": "randomized-nested-queries",
+		"operation-type": "search",
+		"param-source": "nested-query-source",
+		"cache": false
+	}],
+	"challenges": [{
+		"name": "nested-search-challenge",
+		"description": "Indexes document corpus for an hour using Elasticsearch default settings . After that randomized nested queries are run.",
+		"default": true,
+		"index-settings": {
+			"index.number_of_replicas": 0
+		},
+		"schedule": [{
+			"operation": "index-append",
+			"warmup-time-period": 20,
+			"time-period":3600,
+			"clients": 4
+		}, {
+			"operation": "force-merge",
+			"clients": 1
+		}, {
+			"operation": "randomized-nested-queries",
+			"clients": 2,
+			"warmup-iterations": 10,
+			"iterations": 5000
```

Comment:
You are (at worst) driving the system into 100% utilization measuring latency at an unstable point. I recommend that you define a fixed throughput (say 50 operations / second). See http://esrally.readthedocs.io/en/latest/faq.html#what-does-latency-and-service-time-mean-and-how-do-they-related-to-the-took-field-that-elasticsearch-returns for details (I also linked to a talk of mine where I explain this topic in more depth although I improved the slides in the mean time, see http://daniel.mitterdorfer.name/talks/2016/Benchmarking%20Elasticsearch.pdf - starting with slide 27)